### PR TITLE
 Add /sbin and /usr/sbin to PATH if not present for archlinux compatibility

### PR DESCRIPTION
**Pull request by skylt**
Enables the use of bootstrap-vz on Archlinux and other platforms not having /sbin and /usr/sbin in their PATH.

### CODE REVIEW

File: bootstrapvz/base/main.py [hunk 1]
```diff
@@ -22,6 +22,10 @@ def main():
     from manifest import Manifest
     manifest = Manifest(path=opts['MANIFEST'])
 
+    # Set-up Architecture independent path
+    if "usr/sbin" not in os.environ["PATH"]:
+        os.environ["PATH"] += os.pathsep + "/usr/sbin" + os.pathsep + "/sbin" + os.pathsep + "/bin"
```
Comment (GabrielL) [inline bootstrapvz/base/main.py]: you could do something simpler, like:

```
os.environ['PATH'] += os.pathsep.join(["/usr/sbin", "/sbin", "/bin"])
```

Comment (GabrielL) [inline bootstrapvz/base/main.py]: I would rewrite the commit message to something like:

> complete PATH for distribution that only have "/usr/bin" in PATH
>
> debian installations expect to have a path that contains /bin, /sbin and /usr/sbin in their path, but some distributions (Archlinux for example) have all these directories merged into /usr/bin.

**Comment by skylt** [comment]
If this is okay I'll squash the commits and rewrite the message.

**Comment by GabrielL** [comment]
Ok, LGTM

File: bootstrapvz/common/tasks/apt.py [hunk 2]
```diff
@@ -193,8 +193,8 @@ class AptUpdate(Task):
 
     @classmethod
     def run(cls, info):
-        log_check_call(['chroot', info.root,
-                        'apt-get', 'update'])
+        log_check_call_chroot(['chroot', info.root,
+                               'apt-get', 'update'])
```
Comment (GabrielL) [inline bootstrapvz/common/tasks/apt.py:197]: why not
```
log_check_call_chroot(info.root, ['apt-get', 'update'])
```

Comment (skylt) [reply bootstrapvz/common/tasks/apt.py:197]: Would there be any use to call something before chroot ?

**Pull request closed without merging**